### ISSUE TITLE
Add contributors.md or something?

### ISSUE THREAD

**Issue by apoelstra** [rust-bitcoin/rust-bitcoin#447]
It would be nice to have a list of people with merge access to this repo (and to other rust-bitcoin repos) so people know who to bug when they're blocked on getting PRs merged.

Would also be good to have some contributor guidelines so people know they're not committing some sort of faux pas.

**Comment by stevenroose** [comment]
I wanted to give this a shot but noticed I can't even see the project settings and thus contributor list.

**Comment by ariard** [comment]
We have a [rust-lightning/CONTRIBUTING.md](https://github.com/rust-bitcoin/rust-lightning/blob/master/CONTRIBUTING.md) for inspiration and staying style consistent across repos.

**Comment by Kixunil** [comment]
Hmm, this was closed but the list of contributors was not added. Is this intended?

**Comment by dr-orlovsky** [comment]
I think this was mistakingly attached to COBTRIBUTING.md PR

**Comment by dr-orlovsky** [comment]
Actually not mistakingly. That was exactly the purpose of that PR: contribution guidelines and list of maintainers (it is there in that file)

**Comment by Kixunil** [comment]
My point is the list is not published anywhere I can find.

**Comment by dr-orlovsky** [comment]
> It would be nice to have a list of people with merge access to this repo (and to other rust-bitcoin repos) so people know who to bug when they're blocked on getting PRs merged.

https://github.com/rust-bitcoin/rust-bitcoin/blob/master/CONTRIBUTING.md#repository-maintainers

> Would also be good to have some contributor guidelines so people know they're not committing some sort of faux pas.

https://github.com/rust-bitcoin/rust-bitcoin/blob/master/CONTRIBUTING.md

So, @Kixunil this issue is indeed closed and addressed. If you need to add some other information, this will be another issue and another PR

**Comment by Kixunil** [comment]
Ah, why didn't I see it? :sweat_smile: Sorry about the noise.

**Comment by dr-orlovsky** [comment]
The list of maintainers has actually to be updated with you, @RCasatta and @tcharding (AFAIR @apoelstra already added him) - so we need another issue or PR to address that.

And maybe having separate MAINTAINERS.md file will also be good to have.

**Comment by apoelstra** [comment]
We are also missing @Kixunil from the list in the .md.
